bigquery/cloud-client/client_secrets.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"1071053816718-pk867pkh36eb1ijd1lembvf54gqbces9.apps.googleusercontent.com","project_id":"bigquery-sample-166321","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://accounts.google.com/o/oauth2/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"skisPURyX9AunQJ9Bd_sW7al","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
Removed.

Not needed after I changed the tests to mock out user credentials. I'll make sure the docs say to put such a file here.